Tribler/Core/DownloadConfig.py

```diff
@@ -193,8 +194,8 @@ def load(filename):
         dlconfig = CallbackConfigParser()
         try:
             dlconfig.read_file(filename)
-        except:
-            raise IOError, "Failed to open download config file"
+        except (ParsingError, IOError):
+            raise IOError, "Failed to open download config file: %s" % filename
```

Comment:
Why not raise the exception that is being thrown here? It might be confusing to debug if a `ParsingError` is raised but an `IOError` is displayed to the dev instead.